### PR TITLE
add tag support to dogwrap

### PR DESCRIPTION
**Pull request by dominis**
Usage:
`dogwrap -n test-job -k $API_KEY --tags "test, cron, test2" --submit_mode all "ls -lah"`

### CODE REVIEW

**Comment by yannmh** [comment]
Great feature, thanks a lot @dominis.

We'll review your PR shortly.

**Comment by gnarf** [comment]
:+1: I wish this had this already!

I'm not the most proficient python programmer, but perhaps you could also submit a fix for #88 - it seems super easy, I'm just not sure what the opposite of the `.extend` method is :)

**Comment by Zemmiph0bia** [comment]
+:1 Awesome!

@yannmh do you know when this will be merged into master?

**Comment by yannmh** [comment]
Sincere apologies for the delay on this.
I'll take a pass on it today, so it'll be merged this week :rocket:

File: datadog/dogshell/wrap.py [hunk 1]
```diff
@@ -189,6 +189,8 @@ def main():
     parser.add_option('-b', '--buffer_outs', action='store_true', dest='buffer_outs', default=False,
                       help="displays the stderr and stdout of the command only once it has \
 returned (the command outputs remains buffered in dogwrap meanwhile)")
+    parser.add_option('--tags', action='store', dest='tags', default=False,
+                      help="comma separated list of tags")
```
Comment (yannmh) [inline datadog/dogshell/wrap.py]: I think setting the `default` to `default=''` would be more consistent.

Nitpicky: can we also set the `type` to `type=string` ?  Even if it's the default value (https://docs.python.org/2/library/optparse.html#optparse.Option.type), I think it makes things clearer :smile:

File: datadog/dogshell/wrap.py [hunk 2]
```diff
@@ -239,6 +241,11 @@ def main():
     if notifications:
         event_body.extend([u'notifications: %s\n' % (notifications)])
 
+    if options.tags is not None:
```
Comment (yannmh) [inline datadog/dogshell/wrap.py]: I don't think `options.tags` can take the `None` value. I would simply check
```python
if  options.tags:
   do_something()
```
to filter out `None`, `False` or `''`

**Comment by yannmh** [comment]
Looking great, thanks for the contribution @dominis !

I just added two nitpicks, otherwise it's ready to be merged.

**Comment by dominis** [comment]
@yannmh i've fixed the things you've mentioned

**Comment by yannmh** [comment]
Thank you !

A new version of `datadogpy` will be out soon, with your changes.

**Pull request merged**